### DISCUSSION TITLE
absence of copilot on visual studio based on arch.

### DISCUSSION THREAD

**Discussion by lolrida** [community/community#14199] [Copilot Conversations]
Hello. I wanted to install copilot on visual studio, but when i tried to search in visual extensions it can't find it. From winwdows he finds it for me, on arch no! Can someone help me?
Thanks.

**Comment by jason-scheffel** [comment]
That is most likely because you are using the OSS version. This is a link to the arch documentation on vscode: 
https://wiki.archlinux.org/title/Visual_Studio_Code

Visual Studio Code — Microsoft-branded binary release.
Is the section where you should be looking at and you will get two options:

visual-studio-code-bin
and 
visual-studio-code-insiders-bin

Pick one to install through aur 

This is the likely issue like I said, so it may not work for you--but a suggestion.

**Reply by Marychoffel22** [reply]
Pull requests

**Comment by Marychoffel22** [comment]
Assigned

**Reply by Marychoffel22** [reply]
Duplicate

**Reply by Joelsamijon** [reply]
👍
